### ISSUE TITLE
Bumpy aprons at several airports

### ISSUE THREAD

**Issue by evilstiefel** [kubilus1/autoortho#117]
Unfortunately, a lot of airports using autoortho mesh have quite bumpy/hilly aprons and weird terrain glitches. Should we list those or is the recommended approach to just create custom tiles for these areas?

If so, is it possible to also just create tiles without textures and use them alongside the default tile package?

Examples are EDDK, EDDM and EDDF, but I'm sure there's a lot more of them.

**Comment by jonaseberle** [comment]
You could go off this Ortho4XP config https://github.com/kubilus1/autoortho-scenery/blob/main/Ortho4XP.cfg and ply with the values - that's AFAIK the config that has been used to generate the autoortho tiles. If you find something that works better for most airports, I am sure this will be thankfully accepted.

**Comment by evilstiefel** [comment]
My problem is: if I create tiles with Ortho4XP and the skip-downloads flag enabled, X-Plane crashes on loading because of missing textures, even if autoortho is running (so same behavior as if I did not have the tool running in the background. I don't know what else is necessary so that autoortho feels "responsible" to download and create the textures.

**Comment by evilstiefel** [comment]
I've found a solution: `apt_smooting_pix` needs to be set from the default value of `8` to something higher, like `32` in my case. This completely solves the issue with too much mesh elevation variance across airport terrain.
You can use the tiles generated by Ortho4XP, just make sure you enable `skip_downloads` and `skip_converts` and you are putting the custom tiles higher in the `scenery_packs.ini` than the tiles from autoortho.

Additionally, you need to create a junction from the folder `textures` insude z_autoortho to every tile you create using Ortho4XP.

**Comment by evilstiefel** [comment]
Just to illustrate the effect, here's apron 8 in EDDM looking west, first the mesh by autoortho, then with apt_smoothing_pix set to 32:

<img width="1068" alt="image" src="https://user-images.githubusercontent.com/8029370/236620358-30de66f6-da1f-404e-bea2-ea0e39c6fe94.png">

<img width="1068" alt="image" src="https://user-images.githubusercontent.com/8029370/236620371-391048e6-31c8-4bc0-9ad7-09e9c03a3c48.png">

**Comment by jonaseberle** [comment]
I've experienced that as well in EDDF and EDDM. But since using better elevation data (https://sonny.4lima.de/) I have not had that problem any more. And that LiDAR mesh is just so much better in many other places as well.

That is with the default `apt_smoothing_pix=8` and Sonny's LiDAR DTM 1":
![image](https://user-images.githubusercontent.com/1678001/236946610-3547628e-02ca-4e35-ae66-5388493163fc.png)

A hack to set that up easily is to unpack all SONNY's LiDAR (I used 1") files into `./Elevation_data`, then run this from there:
```
#! /bin/bash
for y in {-9..9}; do  
  for x in {-18..18}; do
    ln -snr ./ "./$(printf "%+d0%+03d0" "$y" "$x")"
  done
done
```
That sets up symlinks so that Ortho4XP will find the files under expected path.

So I don't know if using that for Europe would be feasible for autoortho. Of course it would need Sonny's permission.

I agree that a higher `apt_smoothing_pix` can't hurt but 32 makes them so perfectly flat I fear that we are losing some fidelity in places where even the default mesh has it. Does 16 or 24 yield good effects, too?

**Comment by evilstiefel** [comment]
I'm not exactly sure what the cut-off point with the best compromise is. When dealing with this issue, I simply remembered what I did years ago when I created my own original Ortho. At that point, I landed at a value of 32px and stuck with it.

This still gives me sloped runways (which is what I mostly care about) and removes any weird glitches. There may be a middle ground.

How did you deal with the water masks? I didn't mention it above, but I've since found out that I need to copy all files from the `textures` folder from the generated Ortho-tiles into `_textures` in `z_autoortho` before deleting the folder and creating the junction, otherwise, some linked masks from the generated `DSF` files are missing and X-Plane might crash or complain about incomplete scenery.

**Comment by kubilus1** [comment]
Will setting apt_smoothing_pix have negative consequences as a general setting for mountain ranges and other terrain that should be 'craggy'?

**Comment by evilstiefel** [comment]
Nope, it'll only affect airports, much like the higher zoomlevel settings, only even more localized

**Comment by kubilus1** [comment]
So updating the Europe scenery with apt_smoothing_pix=32.  No real big increase in size, which is good.  However, not sure if this is a coincidence or not, I'm seeing some issues with the mesh that weren't there before.  Here's just south of Zurich:

![CirrusSF50 - 2023-05-28 16 19 12](https://github.com/kubilus1/autoortho/assets/6963628/b2da6bd9-c9b4-4d27-b48b-d4a10e40f4b9)

**Comment by evilstiefel** [comment]
I haven't observed such issues with changing that setting, not sure what's going on there

**Comment by bobsleigh44** [comment]
Hi, @kubilus1 Just wondering if you ended up updating the scenery with the smoother mesh files because the airports still seem quite hilly.
